### PR TITLE
[UI Component] Adding custom alerts

### PR DESCRIPTION
**Pull request by carlosvalle**
## Description

Adds a way to add custom alerts
## Format

```
$usa-custom-alerts: (
  Name of alert 1 : Background Color ,
  Name of alert 2 : Background Color
);
```
### my-variables.scss

```
$usa-custom-alerts: ( 
  note: $backgroundcolor-note,
  message: $backgroundcolor-message
)
```

Before you hit Submit, make sure you’ve done whichever of these applies to you:
- [x] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [x] Run `npm test` and make sure the tests for the files you have changed have passed.
- [x] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

**Comment by msecret** [comment]
@carlosvalle thanks for submitting! Could you help us out a little by posting a screenshot of what some of your custom alerts look like?

File: src/stylesheets/components/_alerts.scss [hunk 1]
```diff
@@ -1,3 +1,17 @@
+// Alert variables ---------- //
+// scss-lint:disable TrailingWhitespace
```
Comment (msecret) [inline src/stylesheets/components/_alerts.scss]: Is there a reason you have to disable trailing whitespace?

Comment (carlosvalle) [inline src/stylesheets/components/_alerts.scss]: $usa-alerts variable for is giving me that specific lint error - I just feel that keeping this format is much more readable

**Comment by carlosvalle** [comment]
alert-note

File: src/stylesheets/components/_alerts.scss (hunk 1 above)
Comment (juliaelman) [inline src/stylesheets/components/_alerts.scss]: @carlosvalle we are no longer using `scss-lint` so this line should be a-okay to remove.

See https://github.com/18F/web-design-standards/commit/95d626ad346748a71c9260514fae0771d83e22d3 for details on those changes.

File: src/stylesheets/components/_alerts.scss [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+
+$usa-alerts: ( 
+  success: $color-green-lightest,
+  warning: $color-gold-lightest,
+  error: $color-secondary-lightest,
+  info: $color-primary-alt-lightest
+);
+
+$alerts: map-merge($usa-alerts, $usa-custom-alerts);
+// scss-lint:enable TrailingWhitespace
```
Comment (juliaelman) [inline src/stylesheets/components/_alerts.scss]: Please remove.

**Comment by carlosvalle** [comment]
@juliaelman - comments removed

**Comment by juliaelman** [comment]
🚀

**Pull request merged**